docs/en/reference/pipeline-integrations/toolkit-engines/tk-create.md

```diff
@@ -45,4 +45,12 @@ If you have access to the {% include product %} Pipeline Toolkit, you also have
 
 ## Special Requirements
 
-You need {% include product %} Pipeline Toolkit Core API version v0.19.1 or higher to use this.
\ No newline at end of file
+You need {% include product %} Pipeline Toolkit Core API version v0.19.1 or higher to use this.
+
+## Fixing {% include product %} Integration Error
+
+Your users are getting the following error message: *Failed to initialize integrations. ValueError - Invalid localhost certificate data in {% include product %}! Please contact support.* This error message indicates that they are trying to access SGTK features when your {% include product %} local install has not enabled SGTK integration. 
```

Comment:
I would say something like your SG instance is not configured to use the SGTK integration.
By default, it's not configured to use toolkit.